apps/full-site-editing/package.json

```diff
@@ -92,6 +92,7 @@
 		"prepare": "check-npm-client && yarn run sync:newspack-blocks --nodemodules"
 	},
 	"dependencies": {
+		"@automattic/calypso-analytics": "^1.0.0-alpha.1",
```

Comment:
I still don't know why we don't just use `*` for these internal packages but it's fine for this PR, I think I'll ask about it in separate PR/issue. :-)